game/screen_sing.hh

```diff
@@ -103,6 +104,9 @@ class ScreenSing: public Screen {
 	std::string m_selectedTrackLocalized;
 	ConfigItem m_vocalTracks[AUDIO_MAX_ANALYZERS];
 	ConfigItem m_duet;
+	size_t players() const { 	boost::ptr_vector<Analyzer>& analyzers = m_audio.analyzers(); return (analyzers.empty() ? 1 : analyzers.size()); } // Always have at least one player to display lyrics and prevent crashes.
```

Comment:
hmm `boost::ptr_vector<T>` can be, and imho should be, `std::vector<std::unique_ptr<T>`. However this would also mean `m_audio.analyzers()` return type need to be `std::vector<std::unique_ptr<T>`

Reply:
That's an enhancement, I guess? I just refactored what was already there.

Reply:
Agreed therefore not blocking it

Reply:
BTW, what with this branch and the search one adding several new config items... I'm thinking I want to reorganize options in the menu. "Game/" has gradually been becoming "Misc"...

We could add an Interface section or something along those lines instead, perhaps?

Reply:
That's pretty neat :+1:

Reply:
Seems like you could just use `auto&` rather than being explicit about the type.